README.md

```diff
@@ -53,28 +77,24 @@ After following the steps, run the command below and follow the instructions to
 ts-node src/google/create-token.ts
 ```
 
-## Deployment
+## Development
 
-Before deployment, create an `.env` file using the fields mentioned below:
+To add new features to the bot. Fork the repository and add your feature to `src/meetbot/features`. After finishing your changes, [run the bot locally](#running-the-bot-locally) to check if your feature loads and test your changes. 
 
-```
-GOOGLE_PASSWORD=              # Password for your bot/user that is used to join Google Meet. 
-GOOGLE_TOTP_SECRET=           # [Optional] If the bot account has 2FA auth, then add 2FA secret here.
-GOOGLE_LOGIN=                 # Email-ID/username of your bot/user
-TOKEN_PATH=token.json         # [Default] Path to token file 
-HTTP_PORT=80                  # Port to run express server on. For prod, use port 80.
-RUNNING_ALPINE=0              # [Default] If base image is alpine then set the env var, otherwise don't. 
-```
+To develop the UI, run `npm run dev-ui`. This will spin up the development server. You should then be able to access the dashboard at `http://localhost:3000/`. Note that the data you will see on the tables are actually being mocked by MirageJS, and not actual request is going to the API. This makes it easier to develop both components in an entirely decoupled way.
 
-Run the command below and follow the instructions to generate a `token.json` file. This authentication process is one time only.
+## Deployment
 
-```
-ts-node src/google/create-token.ts
-```
+To activate all the features of meetbot, [authenticate the meetbot](#authentication) and create the `token.json` file before deployment. 
 
-To deploy, use the dockerfiles present in the repository. Tweak the base image and other details as per your deployment platform (AWS, Heroku etc.) and deploy. 
-To deploy on balenaCloud, run the command below:
+To deploy on balenaCloud, use the balena CLI and balena push command as stated below. For more information [check out the docs](https://www.balena.io/docs/learn/deploy/deployment/).
```

Comment:
Is it just me or is "belenaCloud" make it sound like this would become a cloud deployment?

Reply:
I focused on productization but kept/left the platform choice biased since it becomes easier for us.

Reply:
Oh yes, that definitely makes sense. That comment was meant in the same way as the one above. If someone doesn't know about the Balena terminology, then "deploy on balenaCloud" (shouldn't it be "deploy with"?) makes it sound like it would be a cloud hoster.
TBH I don't know a better wording that uses the correct branding and is unambiguous...
Maybe
```suggestion
To deploy to your device with the help of [balenaCloud](https://www.balena.io/cloud/), use [balena CLI](https://github.com/balena-io/balena-cli/blob/master/INSTALL.md)'s `balena push` command as stated below. For more information check out the [balena docs](https://www.balena.io/docs/learn/deploy/deployment/).
```